Readme.md

```diff
@@ -50,4 +79,14 @@ export LDFLAGS="-L/opt/homebrew/opt/openssl@1.1/lib -L/opt/homebrew/opt/libpq/li
 export CPPFLAGS="-I/opt/homebrew/opt/openssl@1.1/include -I/opt/homebrew/opt/libpq/include"
 
 pipenv install
-```
\ No newline at end of file
+```
+
+### Docker
+
+Build command `docker build -t insure .`
+
+## Code Review Process
+
+1. Create and push a new branch
+2. Make sure tests pass, frontend: `npm run test` api: not configured yet
```

Comment:
Lets nix 2 altogether. There are no frontend tests, the backend tests are automatically run via github actions. 

Or maybe we can just say: run backend tests `python manage.py test`